social-network/src/components/Article.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './Article.css'
+import ArticleImage from './ArticleImage';
+import Summary from './Summary';
+import DetailBlock from './DetailBlock';
+import './ArticleImage.css';
+import PropTypes from 'prop-types';
+
+
+class Article extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className={`article-grid__container-${this.props.articleType}`}>
+        <div className={ `article d-flex flex-${this.props.articleType}` } onClick={ () => window.open(this.props.article.link, '_blank')}>
```

Comment:
Another way to do this would be to use an anchor tag (`a`) and set `target="_blank"` and `href={this.props.article.link}`. Both work though.